File: carrinho_de_compras/2018-1/elisson C#/Produto.cs
using System.Collections;
using System.Collections.Generic;

public class Produto {
	public string Nome { get; set; }
	public float Preco { get; set; }
	public Produto(string nome, float preco) {
		this.Nome = nome;
		this.Preco = preco;
	}
}

File: carrinho_de_compras/2018-1/eduardo java/Produto.java
public class Produto {
	
	private String nome;
	private float preco;

	public Produto(){
		
	}

	public Produto(String nome, float preco){
		this.nome = nome;
		this.preco = preco;
	}

	public String getNome(){
		return this.nome;
	}
	public void setNome(String nome){
		this.nome = nome;
	}
	public float getPreco(){
		return this.preco;
	}
	public void setPreco(float preco){
		this.preco = preco;
	}
}